src/mockServer/msw/handlers.ts

```diff
@@ -46,6 +46,7 @@ const getMemberIdFromToken = (bearer: string | null): string => {
 export const buildMSWMocks = (
   { apiHost }: LocalContext,
   database?: Database,
+  appContext?: LocalContext,
```

Comment:
Wouldn't it be clearer if buildMSWMocks ask apiHost and appContext without apiHost ?
It avoids a possible confusion when seeing something like `buildMSWMocks(context, db, context, 'db name');`.

With these changes: 
```
const buildMSWMocks = (
  apiHost: string,
  database?: Database,
  appContext?: Omit<LocalContext, 'apiHost'>,
  dbName?: string,
)
```

you can then call the function with less confusion ? `buildMSWMocks('host', db, context, 'db name');`

If I missed or misunderstand something, just pass this comment.